### PR TITLE
feat: add option to change api description and version

### PR DESCRIPTION
**Pull request by achamwada**

### CODE REVIEW

File: src/ServerlessAutoSwagger.ts [hunk 1]
```diff
@@ -39,6 +39,7 @@ export default class ServerlessAutoSwagger {
     swagger: '2.0',
     info: {
       title: '',
+      description: '',
```
Comment (bfaulk96) [inline src/ServerlessAutoSwagger.ts:42]: I would think it would be safer to default to no description, right?
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -64,6 +64,8 @@ custom:
 | `swaggerFiles`            | Array of string which will merge custom json OpenApi 2.0 files to the generated swagger                               | `[]`                                                                       |                                                          |
 | `swaggerPath`             | String for customize swagger path. Your new swagger UI will be available at `https://{your-url-domain}/{swaggerPath}` | `swagger`                                                                  | `my-swagger` => `https://{your-url-domain}/my-swagger`   |
 | `title`                   | String to overwrite the project title with a custom one                                                               | Serverless service name                                                    |                                                          |
+| `description`             | String to overwrite the project description with a custom one                                                         | Serverless service description                                             |                                                          |
```
Comment (bfaulk96) [inline README.md:67]: For default, `- `

File: README.md [hunk 3]
```diff
@@ -64,6 +64,8 @@ custom:
...
+| `version`                 | String to overwrite the project version with a custom one                                                             | Serverless service version                                                 |                                                          |
```
Comment (bfaulk96) [inline README.md:68]: For default, `1`

**Review by bfaulk96 — CHANGES_REQUESTED** [review]
Looks good to me other than those 3 small changes!

Thanks for contributing!

File: README.md (hunk 2 above)
Comment (achamwada) [reply README.md]: Updated

**Review by bfaulk96 — APPROVED** [review]

**Comment by bfaulk96** [comment]
@achamwada this is released and available in [v2.10.0](https://github.com/completecoding/serverless-auto-swagger/releases/tag/v2.10.0) 😄 Thanks again for the contributions1

**Pull request merged**